docparse/output_walkthrough.mdx

```diff
@@ -0,0 +1,174 @@
+
+## Examining Individual Elements
+
+### Extracting an Image from the Document
+
+```json image.json
+
+{ 
+  type': 'Image',
+ 'bbox': [0.0873676793715533,
+  0.08911955399946733,
+  0.8548237161075367,
+  0.5357984508167614],
+ 'properties': {'score': 0.873895525932312,
+  'image_size': [1324, 1003],
+  'image_mode': 'RGB',
+  'image_format': None,
+  '_element_index': 1,
+  'page_number': 2},
+ 'binary_representation': ...
+}
+```
+The first element we see is an image. We get back a bounding box and a binary representation of the image, which we can use to display the image.
+
+### Extracting Captions from the Document
+
+```json caption.json
+
+{'type': 'Caption',
+ 'bbox': [0.08805406458237591,
+  0.5466218150745739,
+  0.8386106244255515,
+  0.5634380548650568],
+ 'properties': {'score': 0.6033732891082764,
+  '_element_index': 3,
+  'page_number': 2},
+ 'text_representation': 'Figure 1. Accident airplane as it came to rest (Source: Federal Aviation Administration)\n'}
+```
+The second element we see is a caption. We get back a bounding box and the text content of the caption.
+
+### Extracting Text from the Document
+
+```json text.json
+{'type': 'Text',
+ 'bbox': [0.08857761158662684,
+  0.5770943936434659,
+  0.5373964197495404,
+  0.5928012917258523],
+ 'properties': {'score': 0.589328944683075,
+  '_element_index': 5,
+  'page_number': 2},
+ 'text_representation': 'The wreckage was retained for further examination. \n'}
+```
+The third element we see is a text element. We get back a bounding box and the text content of the element.
+
+### Extracting a Table from the Document
+
+```json table.json
+
+{ 
+ 'type': 'table',
+ 'bbox': [0.09319061279296875,
+  0.6873569003018466,
+  0.9122876694623162,
+  0.8174545010653409],
+ 'properties': {'score': 0.8106631636619568,
+  'title': None,
+  'columns': None,
+  'rows': None,
+  '_element_index': 2,
+  'page_number': 2},
+ 'text_representation': None,
+ 'table': {'cells': [{'content': 'Aircraft Make:',
+    'rows': [0],
+    'cols': [0],
+    'is_header': True,
+    'bbox': {'x1': 0.09705882369281045,
+     'y1': 0.6968232236868688,
+     'x2': 0.27310294133986934,
+     'y2': 0.7097020113257576},
+    'properties': {}},
+   {'content': 'MARC JONES',
+    'rows': [0],
+    'cols': [1],
+    'is_header': True,
+    'bbox': {'x1': 0.31151960800653594,
+     'y1': 0.6968232236868688,
+     'x2': 0.43984150343137257,
+     'y2': 0.7097020113257576},
+    'properties': {}},...
+}
+```
+The fourth  element we see is an table. We get back a bounding box and the table structure, which includes the cells of the table and their properties.
```

Comment:
Might make sense to zoom in on a cell element as well and just walk through it too.

Reply:
Done.